ec2-cluster/variables.tf

```diff
@@ -0,0 +1,83 @@
+variable "aws_region" {
+  type    = string
+  default = "us-west-1"
+}
+
+variable "env" {
+  type    = string
+  default = "dev"
+}
+
+variable "instance_count" {
+  type    = number
+  default = 3
+}
+
+variable "instance_type" {
+  type    = string
+}
+
+variable "node_disk_size" {
+  type    = number
+  default = 50
+}
+
+variable "tessellation_version" {
+  type    = string
+  default = "0.12.0"
+}
+
+variable "instance_keys" {
+  type = list(object({
+    key = string,
+    id  = string
+  }))
+  default = [
+    { key = "key-0.p12", id = "e2f4496e5872682d7a55aa06e507a58e96b5d48a5286bfdff7ed780fa464d9e789b2760ecd840f4cb3ee6e1c1d81b2ee844c88dbebf149b1084b7313eb680714" },
```

Comment:
Maybe we should use Amazon KMS to store these keys?

Reply:
It could make sense. The only caveat is that in KMS won't allow you to import the keystore, so we need to export keys from keystores first (in a compatible format) and then import them to KMS. After that, we need to convert it back to p12 so a node can consume that.

Reply:
Also we need to store `id` together with `p12` so the question is if it is worth it.